README.md

```diff
@@ -333,7 +335,9 @@ You can also check your proposal's validity using the [`validate-cbor`](#validat
 
```

Comment:
Just a suggestion:

If we are giving examples for creating and submitting proposal vote then maybe we can also add similar examples for update proposals with mandatory arguments: 

cardano-cli -- byron create-byron-update-proposal --config configuration/defaults/liveview/config-0.yaml --signing-key configuration/defaults/liveview/genesis/delegate-keys.000.key --protocol-version-major 1 --protocol-version-minor 0 --protocol-version-alt 0 --application-name app --software-version-num 1 --system-tag linux --installer-hash da39a3ee5e6b4b0d3255bfef95601890afd80701 --filepath ProtocolUpdateProposalFile 

and 

cardano-cli -- byron submit-byron-update-proposal --config configuration/defaults/liveview/config-0.yaml --filepath ProtocolUpdateProposalFile --socket-path socket/node-0-socket 

with example output that user should expect
Update proposal TxId: upid:b982f442